iroh-unixfs/Cargo.toml

```diff
@@ -0,0 +1,65 @@
+[package]
+name = "iroh-unixfs"
+version = "0.1.3"
+edition = "2021"
+authors = ["dignifiedquire <me@dignifiedquire.com>"]
+license = "Apache-2.0/MIT"
+repository = "https://github.com/n0-computer/iroh"
+description = "Implementation of unixfs for iroh"
+rust-version = "1.63"
+
+exclude = [
+  "tests/**",
+  "fixtures/**",
+]
+
+[dependencies]
+anyhow = "1"
+async-channel = "1.7.1"
+async-recursion = "1.0.0"
+async-stream = "0.3.3"
+async-trait = "0.1.53"
+base64 = "0.13.1"
+bytes = "1.1.0"
+cid = "0.9"
+fastmurmur3 = "0.1.2"
+futures = "0.3.21"
+iroh-metrics = { version = "0.1.3", path = "../iroh-metrics", default-features = false, features = ["resolver", "gateway"] }
+iroh-rpc-client = { version = "0.1.3", path = "../iroh-rpc-client", default-features = false }
+iroh-util = { version = "0.1.3", path = "../iroh-util", default-features = false }
+iroh-content = { path = "../iroh-content" }
+libipld = "0.15.0"
+libp2p = { version = "0.50", default-features = false, features = ["serde"] }
+multihash = "0.16.3"
+num_enum = "0.5.7"
+once_cell = "1.13.0"
+prost = "0.11"
+rand = "0.8.5"
+reqwest = { version = "0.11", features = ["rustls-tls", "json"], default-features = false }
+serde = { version = "1", features = ["derive"] }
+serde_json = "1.0.87"
+tokio = { version = "1", features = ["fs"] }
+tokio-util = { version = "0.7", features = ["io"] }
+tracing = "0.1.34"
+trust-dns-resolver = { version = "0.22.0", features = ["dns-over-https-rustls", "serde-config", "tokio-runtime"] }
+unsigned-varint = "0.7.1"
+fnv = "1.0.7"
+
+[dev-dependencies]
+criterion = { version = "0.4.0", features = ["async_tokio"] }
+iroh-car = { version = "0.1.3", path = "../iroh-car" }
+iroh-rpc-types = { version = "0.1.3", path = "../iroh-rpc-types", default-features = false }
+iroh-store = { version = "0.1.3", path = "../iroh-store", default-features = false }
+proptest = "1.0.0"
+rand = "0.8.5"
+rand_chacha = "0.3.1"
+tempfile = "3.3.0"
+tokio = { version = "1", features = ["rt", "macros", "rt-multi-thread", "fs"] }
+ruzstd = "0.3"
```

Comment:
I don't think these dependencies are all needed, eg ruzstd is only used in iroh-resolver I think

Reply:
ahh yes, thank you, I meant to do a cargo clean up!